openpype/lib/terminal.py

```diff
@@ -49,8 +49,8 @@ def _initialize():
         """
 
         from openpype.lib import env_value_to_bool
-        use_colors = env_value_to_bool(
-            "OPENPYPE_LOG_NO_COLORS", default=Terminal.use_colors
+        use_colors = not env_value_to_bool(
+            "OPENPYPE_LOG_NO_COLORS", default=not Terminal.use_colors
         )
         if not use_colors:
             Terminal.use_colors = use_colors
```

Comment:
It is right but I had to read it 3 times to understand the logic. I suggest for readibility this change
```suggestion
        log_no_colors = env_value_to_bool(
            "OPENPYPE_LOG_NO_COLORS", default=None
        )
        if log_no_colors is not None:
            Terminal.use_colors = not log_no_colors
        
        if not Terminal.use_colors:
```